### ISSUE TITLE
ActiveRecord::StatementInvalid - PSQL - "people with more than one submission"

### ISSUE THREAD

**Issue by eladeyal-intel** [frab/frab#554]
When clicking Reports --> Query Speakers --> people with more than one submission

if the database is PostgreSQL

You get

```An ActiveRecord::StatementInvalid occurred in reports#show_people:

  PG::GroupingError: ERROR:  column "people.id" must appear in the GROUP BY clause or be used in an aggregate function LINE 1: SELECT DISTINCT "people".* FROM "people" INNER JOIN "event_p...
                        ^
: SELECT DISTINCT "people".* FROM "people" INNER JOIN "event_people" ON "event_people"."person_id" = "people"."id" INNER JOIN "events" ON "events"."id" = "event_people"."event_id" INNER JOIN "conferences" ON "conferences"."id" = "events"."conference_id" WHERE "conferences"."id" = $1 AND "event_people"."event_role" = $2 GROUP BY event_people.person_id HAVING (count(*) > 1)
  app/controllers/reports_controller.rb:121:in `show_people'

```

It works OK with sqlite3

**Comment by eladeyal-intel** [comment]
I guess this needs to be fixed here :
https://github.com/frab/frab/blob/a688e0195019200d16ca9807c4a95145a032aa9b/app/controllers/reports_controller.rb#L91

**Comment by eladeyal-intel** [comment]
Submitted fix in pull#556. I tested this with PostgreSQL and with sqlite3.
